infra/cifuzz/cifuzz.py

```diff
@@ -164,6 +167,27 @@ def run_fuzzers(project_name, fuzz_seconds, workspace):
     else:
       logging.info('Fuzzer %s, detected error: %s.', target.target_name,
                    stack_trace)
-      shutil.move(test_case, os.path.join(out_dir, 'testcase'))
+      shutil.move(test_case, os.path.join(bug_report_dir, 'test_case'))
+      parse_fuzzer_output(stack_trace, bug_report_dir)
       return True, True
   return True, False
+
+
+def parse_fuzzer_output(fuzzer_output, out_dir):
```

Comment:
Was this based on clusterfuzz code at all? ClusterFuzz has solved this problem already so there may be some edge cases it catches that we wont

Reply:
I switched it to use a simplified version of clusterfuzz's method.